src/components/v2/ProgressBarHorizontal/PercentageRepresentation/index.tsx

```diff
@@ -0,0 +1,29 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import Typography from '@mui/material/Typography';
+
+import { IProgressBarProps, ProgressBarHorizontal } from '..';
+import { useStyles } from './styles';
+
+export const PercentageRepresentation: React.FC<Pick<IProgressBarProps, 'value' | 'className'>> = ({
+  value,
+  className,
+}) => {
+  const styles = useStyles();
+  return (
+    <div css={styles.root}>
+      <ProgressBarHorizontal
+        className={className}
+        value={value}
+        mark={undefined}
+        step={1}
+        ariaLabel="percentage"
+        min={0}
+        max={100}
+      />
+      <Typography component="span" variant="small1" color="text.primary" css={styles.percentage}>
+        {value}%
```

Comment:
That's a small thing, but we have a util to display percentages now: `formatToReadablePercentage` (you can import it from `utilities/common`).

Reply:
done